versions.properties

```diff
@@ -32,16 +32,16 @@ version.io.mockk..mockk=1.10.0
 version.io.reactivex.rxjava2..rxandroid=2.1.1
 version.io.reactivex.rxjava2..rxjava=2.2.19
 version.junit.junit=4.13
-version.kotlin=1.3.72
-version.kotlinx.coroutines=1.3.7
-version.kotlinx.serialization=0.20.0
+version.kotlin=1.4.10
+version.kotlinx.coroutines=1.4.1
+version.kotlinx.serialization=1.0.0
```

Comment:
1.0.1 was released 3 weeks ago with few patches

Reply:
This is only used for samples, but better to be up-to-date.